.gitignore

```diff
@@ -1,5 +1,8 @@
+# ignore fastly binary
+#
+# NOTE: This can cause fuzzy finders to fail to locate cmd/fastly/main.go
+# because gitignore cannot negate files inside an ignore directory.
 fastly
-!cmd/fastly
```

Comment:
Just to clarify: I'm removing this line as per the note above (i.e. it didn't actually negate the ignored item).

I also Googled for ways to ignore binaries and unfortunately there's no easy way to do that.